src/inverseRobot.js

```diff
@@ -8,6 +8,16 @@
 
 function inverseRobot(robot) {
```

Comment:
It's a good practice to add a function description using JSDoc comments. This helps other developers understand what the function does, its parameters, and its return value.